json_schema/type/protocol/analysis/analysis_protocol.json

```
@@ -73,6 +73,13 @@
             "type": "string",
             "user_friendly": "Alignment software version",
             "example": "v2.0.1; 2.4.2a; v0.45.2"
+        },
+        "gene_annotation_version": {
+            "description": "The Ensembl release version accession number or NCBI RefSeq assembly version used for gene annotation.",
+            "type": "string",
+            "pattern": "^v[0-9]{2,}$|^GCF_[0-9]{9}\\.[0-9]{1,}$",
```

Any reason not to use compact `\d` instead of `[0-9]` and `+` instead of `{1,}`?

```suggestion
            "pattern": "^v\\d{2,}$|^GCF_\\d{9}\\.\\d+$",
```